docs/how-to-upload-manifest.md

```diff
@@ -1,107 +1,72 @@
-# How To Push Operator Manifest(s) To App Registry
-This document shows how one can push operator manifest(s) into `quay.io` or any other compatible app registry server.
+# How To Push an Operator Artifact To an App Registry
 
-*The example script(s) in this document refers to quay.io*
+This document shows how one can push an operator artifact onto `quay.io` or any other compatible app registry server. All commands in this document should be run in the root of this repository.
 
-## Format of Operator Manifest
-*As proof of concept, we are storing the operator manifest into one file. It could change in the future.*
+## What is an Operator Artifact?
 
-The operator manifest file has the following structure -
-```yaml
-name: certified operators
-publisher: redhat
+An operator artifact is a collection of data required to make an operator offering available to a cluster. Each operator artifact represents one operator offering.
```

Comment:
Please add verbiage mentioning that you can have multiple operator offerings in a namespace.